docs/configuration.rst

```diff
@@ -75,6 +76,7 @@ Validator List
     validators/headRef.rst
     validators/label.rst
     validators/milestone.rst
+    validators/or.rst
```

Comment:
Thoughts on adding these under advanced operators?

Reply:
you mean a page called `advanced operators?(validator?)` and put these in them?

Reply:
Yes. These are operators. So on the side bar you'd have `Validators`, `Operators` and `Actions`

Reply:
addressed

Reply:
Since you've added the Operators section and listed the operators there, should it still be on the validator list?

Reply:
you are right, it was an oversight, fixed